scheduler/src/cook/mesos/api.clj

```diff
@@ -1552,21 +1557,29 @@
                                                                           (map first)
                                                                           (map str)
                                                                           (into [])))}]
-
                          :else
-                         (let [groups (mapv #(validate-and-munge-group (db conn) %) groups)
-                               jobs (mapv #(validate-and-munge-job
-                                             (db conn)
-                                             user
-                                             task-constraints
-                                             gpu-enabled?
-                                             (set (map :uuid groups))
-                                             %
-                                             :override-group-immutability?
-                                             override-group-immutability?) jobs)]
-                           [false {::groups groups
-                                   ::jobs jobs
-                                   ::pool pool}]))
+                         (let [in-debt? (ratelimit/in-debt!? create-job-ratelimiter user)]
```

Comment:
Instead of making the call to `in-debt!?` can we not make a **single** call to `time-until!?` and check its result to trigger the same logic.

Also, I do not like having to convert the return value to seconds manually. Please either report the value in milliseconds in the log (preferable) or use `(-> value t/seconds t/in-millis)` to perform the conversion.